.github/workflows/slow_ci_remainder.yml

```diff
@@ -0,0 +1,55 @@
+name: Slow CI Reminder
+
+# For security reason, don't use `pull_request` but `pull_request_target`!
+on:
+  pull_request_target:
+    paths:
+      - '**.py'
```

Comment:
we can most probably filter modeling codes here

Reply:
also not sure we want slow tests to run on ALL PR, otherwise we would just activate it by default

Reply:
I would like to hear from @amyeroberts and @gante . But if I understand correctly, changes to generation related file or even soom test files, they both count.

We don't activate it by default is sometimes we don't want to run too many files in a PR slow CI.

And here it's just a remind for reviewers. If they think a PR is not worth a slow CI, we don't need to trigger it.

Reply:
> if we don't have @ydshieh commenting, but a huggy-bot or something!

It's not me commenting 😆 , it would be [HuggingFaceDocBuilderDev](https://github.com/HuggingFaceDocBuilderDev),

Reply:
Yeah, it's a bit of a balancing act here -- just like the fetcher it's very hard to guarantee we'll always capture the minimal set of tests needed to be run and have to accept either some misses or too many selected. 

My preference would be: just trigger this warning for a subset of files we know pretty much ALWAYS need slow model tests: modeling code changes; changes to generation code; changes to attention mask creation. 

For all other cases, it's up to the discretion of the reviewer to ask for slow tests or not.

If there's a set of files we always run slow tests for, then we can always update this to include; or remove paths if it's annoying. That's the great thing about software!

Reply:
@ArthurZucker 

changed it to

```
    paths:
      - "src/transformers/models/*/*.py"
      - "src/transformers/generation/*.py"
      - "src/transformers/pipelines/*.py"
      - "tests/models/*/*.py"
      - "tests/generation/*/*.py"
      - "tests/pipelines/*/*.py"
```